include/atari.h

```diff
@@ -86,14 +90,20 @@
 #define CH_HLINE        0x12
 #define CH_VLINE        0x7C
 
-/* color defines */
 
-/* make GTIA color value */
+/*****************************************************************************/
+/* Color definitions                                                         */
+/*****************************************************************************/
+
+/* Make a GTIA color value */
 #define _gtia_mkcolor(hue,lum) (((hue) << 4) | ((lum) << 1))
 
-/* luminance values go from 0 (black) to 7 (white) */
+/* Luminance values go from 0 (black) to 7 (white) */
 
-/* hue values */
+/* Hue values */
+/* (These can vary depending on TV standard (NTSC vs PAL),
+** tint potentiometer settings, TV tint settings, emulator palette, etc.)
+*/
 #define HUE_GREY        0
 #define HUE_GOLD        1
 #define HUE_GOLDORANGE  2
```

Comment:
I guess these hue values could be moved to `_gtia.h` as you suggested in a recent email to cc65-devel.